packages/features/bookings/components/BookerSeo.tsx

```diff
@@ -38,10 +38,11 @@ export const BookerSeo = (props: BookerSeoProps) => {
   const profileName = event?.profile?.name ?? "";
   const profileImage = event?.profile?.image;
   const title = event?.title ?? "";
+  const description = event?.description ?? "";
   return (
     <HeadSeo
       title={`${rescheduleUid && !!bookingData ? t("reschedule") : ""} ${title} | ${profileName}`}
-      description={`${rescheduleUid ? t("reschedule") : ""} ${title}`}
```

Comment:
This looks like it breaks existing event types; the title should remain used I reckon.

Reply:
Should I add a check to use title for normal events and description for dynamic events?

Reply:
we can keep title here for now